replays/theme/wrapper.inc.template.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+include_once '../../pokemonshowdown.com/config/config.inc.php';
+
 if ((substr($_SERVER['REMOTE_ADDR'],0,11) === '69.164.163.') ||
```

Comment:
```suggestion

if ((substr($_SERVER['REMOTE_ADDR'],0,11) === '69.164.163.') ||
```
oops, that shouldnt be necessary anymore since we're building it